packages/itwinui-react/src/core/Popover/Popover.tsx

```diff
@@ -239,36 +249,46 @@ export const Popover = React.forwardRef((props, forwardedRef) => {
     setPopoverElement,
   );
 
+  const triggerId = `${useId()}-trigger`;
+  const hasAriaLabel = !!props['aria-labelledby'] || !!props['aria-label'];
+
   return (
     <>
       {cloneElementWithRef(children, (children) => ({
+        id: children.props.id || triggerId,
         ...popover.getReferenceProps(children.props),
```

Comment:
Is having an id on `children` required? If so, why?

Reply:
yes, for labelling.